src/Helpers/Configuration.test.ts

```diff
@@ -0,0 +1,51 @@
+import { describe } from 'riteway'
+import { createEnvToConfigurationKeyMap, toPair } from './Configuration'
+
+describe('toPair()', async (should: any) => {
```

Comment:
We don't need to test this `toPair` function, it's just an implementation detail of the main function. Lookup whitebox vs blockbox testing.

Reply:
Eric mentions block box testing in this article a few times and explains it a bit https://medium.com/javascript-scene/mocking-is-a-code-smell-944a70c90a6a

Reply:
The reason why I see it as a implementation detail is that its not general enough to be reused, in another function. Its very specific to the format of the function we are creating if that makes sense?